README.md

```diff
@@ -1,4 +1,21 @@
-# Python Boilerplate For GitHub Tasks
+# Music School
 
 **Please note:** read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md)
-before starting.
+before start.
+
+Welcome to our Music School.
+A lot of musicians apply to us every day and we need you to create model for it named `Musician`.
+
+This model should have such fields:
+* `first_name` — `CharField` with the `max_length` of 64;
+* `last_name` — `CharField` with the `max_length` of 64;
+* `instrument` — `CharField` with the `max_length` of 64;
```

Comment:
I know, that previously I said, that 64 is a good choice, but 63 seems to be more good from the memory prospective.
Sorry for that.